packages/server/lib/cypress.coffee

```diff
@@ -54,12 +61,32 @@ module.exports = {
       else
         new Promise (resolve) ->
           cypressElectron = require("@packages/electron")
+          electronStarted = Number(new Date())
           fn = (code) ->
             ## juggle up the totalFailed since our outer
             ## promise is expecting this object structure
             debug("electron finished with", code)
-            resolve({totalFailed: code})
-          cypressElectron.open(".", require("./util/args").toArray(options), fn)
+            electronFinished = Number(new Date())
```

Comment:
detect display or missing dependency problem by noticing how quickly Electron exits. If we suspect display or missing dependency problem then return a special exit code